app/components/faq/FaqPage.component.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/jsx-no-target-blank */
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import {
+    Segment, Header, Container, Icon
+} from 'semantic-ui-react';
+import QAndASection from './QAndAPair.component';
```

Comment:
Can you import `QAndAPair` as `QAndAPair` instead of `QAndASection` or rename the component to `QAndASection`?